README.md

```diff
@@ -6,9 +6,12 @@
 
 ### Data Helpers
 
-
 ### Serialize
 
+Simplify and standardize writing ArchivesSpace data obtained from the data helpers to an external serialization. Serialization formats should include CSV, TSV, EAD, and METS.
```

Comment:
Since there are endpoints for EAD and METS I think we can dispense with those. I also think we should specify "tabular data" because serialization into XML or JSON or something else is a VERY different thing than creating a spreadsheet.